.env

```diff
@@ -0,0 +1,10 @@
+BOOT_NODE_LOCAL_1=QmWYmZrHFPkgX8PgMgUpHJsK6Q6vWbeVXrKhciunJdRvKZ
+# e.g. local or testnet_latest
+CHAIN_VERSION=local
+# e.g. 127.0.0.1 or testnet-harbour.datahighway.com
+ENDPOINT_DEVELOPMENT=127.0.0.1
+ENDPOINT_TESTNET=testnet-harbour.datahighway.com
+# Development or Production
+NODE_ENV=Development
+NODE_KEY=88dc3417d5058ec4b4503e0c12ea1a0a89be200fe98922423d4334014fa6b0ee
+SENTRY_NODE_TESTNET_1=QmVuryfE427VRqrqqXsGuWpwBk4g8mGXgYmnt3f1v6j78r
```

Comment:
Do we want this none existing sentry node left as an example only as it does not exist?
Also the .env file should be ignored as per .gitignore and .env.sample only kept in the repository.